runtime/onert/backend/trix/DevContext.cc

```diff
@@ -71,7 +71,13 @@ DevContext::~DevContext()
 
 ModelID DevContext::registerModel(const std::string &model_file_path)
 {
-  auto meta = getNPUmodel_metadata(model_file_path.c_str(), false);
+  if (_dev_handles.size() == 0)
+  {
+    throw std::runtime_error("No npu device is available");
+  }
+
+  std::unique_ptr<npubin_meta, decltype(&free)> meta(
+    getNPUmodel_metadata(model_file_path.c_str(), false), free);
```

Comment:
Is it OK if the type is `decltype(&free)` and the argument value is `free`?
Both are prefixed with `&` in [the cpp example](https://en.cppreference.com/w/cpp/memory/unique_ptr).
```c++
using unique_file_t = std::unique_ptr<std::FILE, decltype(&close_file)>;
        unique_file_t fp(std::fopen("demo.txt", "r"), &close_file);
```